examples/grpc-debug/main.go

```diff
@@ -40,7 +40,7 @@ func main() {
 
 	dialOption := clientOptions.DialOption(storj.NodeID{})
 
-	conn, err := grpc.Dial(*targetAddr, dialOption)
+	conn, err := grpc.Dial(*targetAddr, dialOption, grpc.WithInsecure())
```

Comment:
Add a note why it's fine to have `grpc.WithInsecure` here.

Reply:
This should be removed... good catch!